include/libtorrent/torrent_info.hpp

```diff
@@ -450,37 +452,36 @@ namespace libtorrent
 
 		// ``merkle_tree()`` returns a reference to the merkle tree for this
 		// torrent, if any.
-		//
+		std::vector<sha1_hash> const& merkle_tree() const { return m_merkle_tree; }
```

Comment:
I think the makes for worse documentation though. especially functions that belong together, like ``merkle_tree()`` and ``set_merkle_tree()``